commands/team_e2e_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
```

Comment:
Typically we keep 3rd party imports in a separate group. Let's do that here too.

Reply:
Fixed. Thanks for catching that!